docker-compose.yml

```diff
@@ -7,6 +7,8 @@ services:
     env_file: .env
     expose:
       - "5432"
+    ports:
```

Comment:
Please delete this before merging otherwise CI is going to break when there are multiple runs.

We could split out the `docker-compose.yml` files into additional overrides to make this possible instead.

Reply:
That might be pretty useful -- I end up forwarding to run tests then forgetting to unforward pretty frequently

Reply:
#5322

Reply:
#5322